src/Controller/UI/ExternalDownloadLogController.php

```diff
@@ -67,10 +95,10 @@ public function defaultAction(Request $request)
                         $typeId = 'anonymous';
                     }
 
-                    $this->container->get('pelagos.event.log_action_item_event_dispatcher')->dispatch(
+                    $this->logActionItemEventDispatcher->dispatch(
                         array(
                             'actionName' => 'File Download',
-                            'subjectEntityName' => get_class($dataset),
+                            'subjectEntityName' => 'Pelagos\Entity\Dataset',
```

Comment:
This name has to be changed. Plan?

Reply:
I still think it should be an improvement after flex, so that we can test the data and reports more efficiently